library/src/main/java/com/google/android/exoplayer/metadata/Id3Parser.java

```diff
@@ -60,13 +65,46 @@ public boolean canParse(String mimeType) {
         byte[] frame = new byte[frameSize - 1];
         id3Data.readBytes(frame, 0, frameSize - 1);
 
-        int firstZeroIndex = indexOf(frame, 0, (byte) 0);
+        int firstZeroIndex = indexOfEOS(frame, 0, encoding);
         String description = new String(frame, 0, firstZeroIndex, charset);
-        int valueStartIndex = indexOfNot(frame, firstZeroIndex, (byte) 0);
-        int valueEndIndex = indexOf(frame, valueStartIndex, (byte) 0);
+        int valueStartIndex = firstZeroIndex + delimiterLength(encoding);
+        int valueEndIndex = indexOfEOS(frame, valueStartIndex, encoding);
         String value = new String(frame, valueStartIndex, valueEndIndex - valueStartIndex,
-            charset);
+                                  charset);
```

Comment:
Hah. Yeah, for indentation you can normally setup your IDE to automatically fix it to the intended style (more or less). Andudo saw the extra space, not me. I think he probably has the eyes/brain plugin thing; guess you just get good at spotting that kinda thing once you've spent enough time staring at code!